keycloak.yaml

```diff
@@ -76,8 +76,18 @@ advisories:
     - timestamp: 2023-04-09T09:54:02.782617-04:00
       status: not_affected
       justification: vulnerable_code_not_in_execute_path
+  CVE-2023-0264:
+    - timestamp: 2023-04-30T07:15:20.601606-04:00
+      status: not_affected
+      justification: component_not_present
+      impact: The CVE was fixed in a version of keycloak prior to our packaging
```

Comment:
Just curious, was a scanner reporting this CVE? (Normally we don't need to comment on CVEs that were remediated before we started packaging, but curious what's going on here)

Reply:
No, but it started getting press yesterday so I researched it to make sure we weren't affected: https://www.offensity.com/en/blog/user-impersonation-via-stolen-uuid-code-in-keycloak-cve-2023-0264/

Reply:
Got it, cool 👍